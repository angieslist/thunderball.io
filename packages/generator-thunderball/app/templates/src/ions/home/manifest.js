module.exports = {
  name: 'home',
  browser: {
    page: {
      createRoutes: './createRoutes',
      path: '/*',
      injectors: [
        '../../appInjectors',
      ],
    },
  },
};
