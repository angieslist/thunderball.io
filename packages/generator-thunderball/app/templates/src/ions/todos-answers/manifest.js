const basePath = '/todo-answers';

export default {
  name: 'todos-answers',
  browser: {
    page: {
      createRoutes: './createRoutes',
      path: `${basePath}*`,
      injectors: [
        '../../appInjectors',
        './injectors',
      ],
      props: {
        basePath,
      },
    },
  },
  middleware: './api/middleware',
};
