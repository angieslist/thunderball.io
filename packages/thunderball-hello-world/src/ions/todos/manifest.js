const basePath = '/todos';

export default {
  name: 'todos',
  browser: {
    page: {
      createRoutes: './createRoutes',
      path: `${basePath}*`,
      injectors: [
        '../../appInjectors',
        './injectors',
      ],
      props: {
        basePath,
      },
    },
  },
};
