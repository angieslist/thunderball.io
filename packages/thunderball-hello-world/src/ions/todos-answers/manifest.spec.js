import manifest from './manifest';

test('is a properly formatted manifest object', () => {
  expect(manifest).toMatchSnapshot();
});
