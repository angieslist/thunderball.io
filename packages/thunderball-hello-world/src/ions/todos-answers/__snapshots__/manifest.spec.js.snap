// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is a properly formatted manifest object 1`] = `
Object {
  "browser": Object {
    "page": Object {
      "createRoutes": "./createRoutes",
      "injectors": Array [
        "../../appInjectors",
        "./injectors",
      ],
      "path": "/todo-answers*",
      "props": Object {
        "basePath": "/todo-answers",
      },
    },
  },
  "middleware": "./api/middleware",
  "name": "todos-answers",
}
`;
